db/migrate/20220818091943_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.integer :PostsCounter
+
```

Comment:
- [ ] [OPTIONAL] We can follow Ruby naming convention when we are writing variable or data attributes, `PostsCounter` should be `posts_counter`

Reply:
Hi @elisha2kyakpo1, thank you for approving my PR and mentioning this Ruby naming convention issue.